README.md

```diff
@@ -325,75 +375,7 @@ If you're able to calibrate and then use Comma 3X to use the steering wheel (aka
-        - First Continental Radar + Camera setup going and thus first radar controlled ACC vehicle done with. This does not mean longitudinal is controlled by openpilot though.
-        - Not sold in the USA, but is in Australia, Japan, and Europe
-        - Only one guy using it in Japan unfortunately. Help double the population!
-
-  - Wave HSM?
-    - These vehicles don't seem to have a valid key in the memory returned after the exploit payload is run.
-    - `🇹 🇸 🇸 3️⃣` 2023 Corolla Cross Hybrid 🔴
-      - 2023: Known to be not working. Key not in visible memory. Mentioned in Willem's blog post.
-    - `🇹 🇸 🇸 2️⃣` 2024 Toyota Highlander 🔴
-      - 2024: Known to be not working. Key not in visible memory.
-      - 02 ~~bootloader~~
-  - Wave with newer ~~bootloader~~? [(It's not a bootloader version?)](https://discord.com/channels/469524606043160576/905950538816978974/1273746993394487376)
-    - These vehicles don't seem to respond to the current exploit to run arbitrary code to do things like dump memory
-    - `🇹 🇸 🇸 2️⃣` 2022 Tundra 🔴
-      - No known ~~bootloader~~ exploit execution
-      - User ThisGuy has an extra rack on the bench. No known progress.
-      - 04 ~~bootloader~~
-    - `🇹 🇸 🇸 2️⃣` 2021 Venza 🟡?🔴?
-      - Key at least not at the same location as the RAV4 Prime
-      - Brute force efforts to find key location TBD
-      - Has a 02 ~~bootloader~~ though from one sample. Strange for this vintage? Maybe another should try.
-    - `🇹 🇸 🇸 2️⃣` 2024 Rav4 Prime 🟡?🔴?
-      - Key at least not at the same location as other RAV4 Prime
-      - Brute force efforts to find key location TBD
-      - At least code is executed. Unknown what might have changed.
-      - New 02 ~~bootloader~~ seen
-    - `🇹 🇸 🇸 2️⃣` 2024 Sienna 🟡?🔴?
-      - Key at least not at the same location as other RAV4 Prime
-      - Brute force efforts to find key location TBD
-      - At least code is executed. Unknown what might have changed.
-      - New 02 ~~bootloader~~ seen
-   - Notable Unknown
-     - `🇹 🇸 🇸 3️⃣` 2023+ Corolla, JP and US origins.
-       - While users have come by here and there, none have produced usable engineering info or experiences of note.
-       - No one has tried getting things to work on a 2023 US-made Corolla that doesn't appear to have TSK.
-     - `🇹 🇸 🇸 2️⃣` 2023+ GR Yaris
-       - Seems externally similar to Yaris Hybrid?
-
-Note: 🟢 = Working, 🟡 = WIP, 🔴 = Not Working and sometimes a mix.
-
-Note 2: If it's not listed above, then there has been no documented information or attempts.
-
-- History and such can be found below. The most recent history is at the bottom. If you need to catch up, start from where you left off. Feel free to ask questions in Discord.
+History and such can be found below. The most recent history is at the bottom. If you need to catch up, start from where you left off. Feel free to ask questions in Discord.
```

Comment:
This one's kind of dangling. I've considered moving it to the bottom of Cars section but after Cars comes Setup Guide so it didn't match what the sentence says. Not sure what to do on this.

Reply:
Ah yeah this is definitely from an old structure. Feel free to blow it away. Or move it.

Reply:
Yeah I can't really find a good way to keep it. I like that it's inviting and friendly, but Cars already has a few sentences for inviting readers to Discord so I think that's covered. I'll delete it.